### ISSUE TITLE
VA Letters page accessibility issues

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va.gov-team#16246]
## What happened?
Android:
- TalkBack Enabled:  When accessing page, the page opens but nothing is announced.
- View letters button is announced as:  "View letters button button".  Unnecessary info "button" included in label.
- With bluetooth keyboard, I'm getting different responses on Androids.
- Samsung Galaxy S7: Tabbing passes over Title (Letters), "Downloaded documents will...", "If this address is incorrect you may..."

Android:
When there is an address in the Mailing address field of the letters page (where you can choose to go to the edit screen), Android says "minus" before the address number and before the zip code.  Example: "minus" one thousand twenty five for 1025 and "minus" twenty thousand 2 for 20002.

iOS & Android:
- With VoiceOver andTalkBack enabled on iOS and Android, the back button a11yHint is not being read
     - NOTE: Should be updated with [a11y update ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/17185)

## Specs:
- Device: Samsung Galaxy S7
- Device: iPhone XR

## Steps to Reproduce
- Connect bluetooth keyboard to device.
- Enable TalkBack.
- Navigate on VA Letters page using Tab and/or arrow keys.
- Samsung Galaxy S7: Tabbing passes over Title (Letters), "Downloaded documents will...", "If this address is incorrect you may..." and does not read them.

## Desired behavior
Tabbing or using arrow keys should read each section of page.

## Acceptance Criteria
- Tabbing or using arrow keys reads each section of page.

## Bug Severity
- 4 - Low

## Linked to Story
https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/14706

## Screen shot(s) and additional information
- iOS does very well with reading each section of page without skipping.

**Comment by digitalken** [comment]
PASS 3/4/2021 - No (minus) spoken before address number, tabbing will hit letter heading.  Closing.
